video/probe.go

```diff
@@ -0,0 +1,40 @@
+package video
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"gopkg.in/vansante/go-ffprobe.v2"
+)
+
+/*
+type ProbeInfo struct {
+	Format  *ffprobe.Format
+	Streams []*ffprobe.Stream
+}
+
+func (p ProbeInfo) String() string {
+	return fmt.Sprintf("Format: %v \nStreams: %v", p.Format, p.Streams)
+}
+*/
+
+func ProbeFileReader(ctx context.Context, fileReader io.Reader) (*ffprobe.ProbeData, error) {
+	return ffprobe.ProbeReader(ctx, fileReader)
+}
+
+func ProbeFileFromOS(file io.Reader) (*ffprobe.ProbeData, error) {
+	var probeInfo ffprobe.ProbeData
+	probeCtx, probeCancel := context.WithTimeout(context.Background(), 60*time.Second)
+	defer probeCancel()
+
+	// TODO: add HTTP probing (via ffprobe.ProbeURL in addition to ffprobe.ProbeReader)
```

Comment:
Let's raise a ticket for the work and delete the TODO

Reply:
Raised https://github.com/livepeer/catalyst-api/issues/395